### ISSUE TITLE
Getting the @index / @first / @last from a root context

### ISSUE THREAD

**Issue by breenyoung** [Handlebars-Net/Handlebars.Net#107]
Maybe I'm gettin the syntax wrong, but anyone know if when you're in a  nested each loop you can get at the parent's context of @index/first/last in the parent loop?

`{{@each}}`

`Parent index: {{@index}}`

```
{{#each}}

    How to get at parent @index?
    {{../@index}} ?
    {{@root.index}} ?

{{/each}}
```

`{{/each}}`

**Comment by rexm** [comment]
Need to figure out why, `{{../../@index}}` will get you the current parent index. That isn't correct or intuitive, but that's how it seems to work right now.

**Comment by rexm** [comment]
How does handlebarsjs work in this situation?

**Comment by imsedim** [comment]
From [here](http://handlebarsjs.com/builtin_helpers.html)

> Nested each blocks may access the interation variables via depth based paths. To access the parent index, for example, {{@../index}} can be used.

**Comment by 304NotModified** [comment]
Isn't this fixed with #174?
